schema/data/prod/json_schema/emission_intensity.json

```diff
@@ -0,0 +1,80 @@
+{
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "title": "Emissions Intensity Report",
+    "required": [
+      "measurementPeriodStartDate",
+      "measurementPeriodEndDate",
+      "emissionFunctionalUnit",
+      "baselineEmissionIntensity",
+      "targetEmissionIntensity"
+    ],
+    "properties": {
+      "reportDueDate": {
+        "type": "string",
+        "title": "Report Due Date"
```

Comment:
add `defaultValue: null` for all optional fields
type should `type: string[] || null` for optional fields

Reply:
to test--clear an optional field, commit, and go check db to make sure it's null in the db (if the previously committed value is still there, it didn't work)

Reply:
for date widgets, might not need to add defaultValue because the widget handles it, check this